ci/dependencies.sh

```diff
@@ -12,3 +12,7 @@ if [ "$CIRCLE_BRANCH" == "master" ]; then
 else
     source ./nvm/nvm.sh && nvm use ${NODE_VERSION} && npm install
 fi
+
+if [ "$CIRCLE_BRANCH" == "master"] || [ -z "$CIRCLE_TAG" ]; then
+    pip install --user --upgrade awscli
+fi
```

Comment:
@lucaswoj Circle complains with `./ci/dependencies.sh: line 16: [: missing `]'` on this part, and then proceeds to install awscli despite not being on master or having a tag. Not that it takes much time, but would be nice to fix.